9.2/root/usr/bin/run-postgresql

```diff
@@ -16,5 +16,10 @@ fi
 
 set_passwords
 
+if [ -f ${CONTAINER_SCRIPTS_PATH}/post-init ]; then
```

Comment:
I have checked MySQL, but I was not able to find anything in it's README. Are you sure that it's documented?